tests/components/zwave_js/conftest.py

```diff
@@ -31,6 +31,12 @@ def multisensor_6_state_fixture():
     return json.loads(load_fixture("zwave_js/multisensor_6_state.json"))
 
 
+@pytest.fixture(name="legacy_binary_sensor_state", scope="session")
+def legacy_binary_sensor_state_fixture():
+    """Load the legacy_binary_sensor node state fixture data."""
+    return json.loads(load_fixture("zwave_js/legacy_binary_sensor_state.json"))
```

Comment:
Please name the fixture file after the device name and not after what we want to test.

Reply:
done